src/loader/bgw_launcher.c

```diff
@@ -66,18 +66,7 @@ typedef struct DbHashEntry
 } DbHashEntry;
 
 
-/*
- * Aliasing a few things in bgWorker.h so that we exit correctly on postmaster
- * death so we don't have to duplicate code basically telling it we shouldn't
- * call exit hooks cause we want to bail out quickly - similar to how the
- * quickdie function works when we receive a sigquit. This should work
- * similarly because postmaster death is a similar severity of issue.
- * Additionally, we're wrapping these calls to make sure we never have a NULL
- * handle, if we have a null handle, we return normal things.  I have left the
- * capitalization in their format to make clear the relationship to the
- * functions in bgworker.h and that these are just passthroughs and should
- * maintain the same behavior unless BGWH_POSTMASTER_DIED is returned.
- */
+/* Handle postmaster death consistently*/
```

Comment:
The new comment doesn't contain quite enough information to be useful to new reader: consistently with what? If we don't want the old comment perhaps something like "On postmaster death we want to bail quickly since no progress is going to be made, and the database restarted."

Reply:
Or just reference the comment in `get_background_worker_pid`

Reply:
That comment was just moved to later, because it was referring to functions further down in the file.